### ISSUE TITLE
A question, why do I get this error?

### ISSUE THREAD

**Issue by DorabaeMC** [dmulloy2/ProtocolLib#3307]
- [ ] This issue is not solved in a development build

**Describe the bug**
Basically every time I die or do a command, for example /effect kicks me out of the server.

**Expected behavior**
Basically, doing that type of command or dying doesn't take me off the server.

**Screenshots**
![imagen_2024-11-10_001121379](https://github.com/user-attachments/assets/14b1e260-cafe-425b-961c-d31e78a98294)

**Version Info**
[dump-2024-11-10_03.12.55.txt](https://github.com/user-attachments/files/17690026/dump-2024-11-10_03.12.55.txt)
**Additional context**
Basically that would be my whole problem, thanks for reading.
